package.json

```diff
@@ -8,6 +8,7 @@
     "@webassemblyjs/ast": "1.4.3",
     "@webassemblyjs/wasm-edit": "1.4.3",
     "@webassemblyjs/wasm-parser": "1.4.3",
+    "@webassemblyjs/wast-loader": "^1.5.4",
```

Comment:
To avoid duplicated deps (with other versions), we should align these, but it's already done by some PR i think.

Reply:
Should be in devDependencies instead

Reply:
Yes, if we move it from the default options